app/extension/bamboo/extension_ui.py

```diff
@@ -0,0 +1,43 @@
+import random
+
+from selenium.webdriver.common.by import By
+
+from selenium_ui.base_page import BasePage
+from selenium_ui.conftest import print_timing
+from util.conf import BAMBOO_SETTINGS
+
+
+def app_specific_action(webdriver, datasets):
+    page = BasePage(webdriver)
+    if datasets['custom_pages']:
+        app_specific_page_id = datasets['custom_page_id']
```

Comment:
What is `custom_page_id` there? Lets remove all `extenstion_ui.py` from this PR and create an appropriate task for this logic.